packages/eva/mapping.json

```diff
@@ -1590,6 +1590,191 @@
         }
       }
     },
+    "Menu": {
+      "meta": {
+        "scope": "all",
+        "parameters": {
+          "dividerHeight": {
+            "type": "number"
+          },
+          "dividerBackgroundColor": {
+            "type": "string"
+          }
+        },
+        "appearances": {
+          "default": {
+            "default": true
+          },
+          "divider": {
```

Comment:
I guess it would be better to control this with component properties like `showDivider={true}`
Otherwise, it would be better to rename it. E.g `noDividers` (to make it consistent with other components like `BottomNavigation` that also could be styled with `appearance='noIndicator`)

Reply:
Done